packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -1522,6 +1526,24 @@ export class API extends EventEmitter {
     );
   }
 
+  getPayProV2(txp) {
+    if (!txp.payProUrl || this.doNotVerifyPayPro) return Promise.resolve();
+
+    const chain = txp.coin ? txp.coin.toUpperCase() : 'BTC';
+
+    return PayProV2.selectPaymentOption(
+      {
+        paymentUrl: txp.payProUrl,
+        chain,
+        currency: '' // TODO
```

Comment:
Use `txp.coin.toUpperCase()` for currency. For ERC20 tokens, the `txp.coin` will be the token symbol, and chain will be `'ETH'`